src/api.c

```diff
@@ -610,7 +610,7 @@ API int seccomp_export_bpf(const scmp_filter_ctx ctx, int fd)
 	rc = write(fd, program->blks, BPF_PGM_SIZE(program));
 	gen_bpf_release(program);
 	if (rc < 0)
-		return -errno;
+		return -ECANCELED;
```

Comment:
I'm wondering if we should consider adding a log/debug feature that could capture the `errno` data we're now throwing away.  It may prove help debugging when a customer files a bug against us.

Reply:
I see your point, I've got an idea, let me see what I can come up with ... stay tuned.

Reply:
As one potential option, I just added a patch to this PR, "api: add the SCMP_FLTATR_API_SYSRAWRC filter attribute".  I'm going to mark this as resolved and we can continue the conversations in the comments or as review comments in the relevant code.